docs/source/reference/types.rst

```diff
@@ -162,12 +162,16 @@ Context                    jit compiled   cfunc compiled   WAP objects
 ========================   ============   ==============   ===========
 Can be used as arguments   yes            yes              yes
 Can be called              yes            yes              yes
-Can be used as items       yes            yes              yes
+Can be used as items       yes1           yes              yes
```

Comment:
Would `yes*` be a more common thing to write? and then below have `* at least one of the...` .